src/mapblock.h

```diff
@@ -552,6 +563,35 @@ class MapBlock /*: public NodeContainer*/
 
 	// Last really changed time (need send to client)
 	std::atomic_uint m_changed_timestamp;
+	u32 m_analyzed_timestamp;
+	u32 m_analyzed_fast_timestamp; //future
+	bool abm_active;
+	typedef std::list<abm_trigger_one> abm_triggers_type;
+	abm_triggers_type * abm_triggers;
```

Comment:
why is that not just std::list<> (not a pointer to std::list) and instead you create/delete it when needed?